autoscale-scaler-marathon/pom.xml

```diff
@@ -54,17 +54,17 @@
             <groupId>com.mesosphere</groupId>
             <artifactId>marathon-client</artifactId>
             <version>0.6.0</version>
-        </dependency>
-        <dependency>
-            <groupId>javax.validation</groupId>
-            <artifactId>validation-api</artifactId>
-            <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>slf4j-simple</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Good catch.  Could we put a PR into https://github.com/mesosphere/marathon-client to fix this properly so that we don't need this exclusion?

Reply:
Issue: 
https://github.com/mesosphere/marathon-client/issues/15

Reply:
fix is on it's way:  https://github.com/mesosphere/marathon-client/pull/16
I'll see if we get the elves to deliver before santa gets here..

Reply:
Removed exclusion for slf4j-simple.

Reply:
You need to move to v0.6.2 to get the fix.